homeassistant/components/ring/light.py

```diff
@@ -76,6 +76,7 @@ def _handle_coordinator_update(self):
             self._attr_is_on = device.lights == ON_STATE
         super()._handle_coordinator_update()
 
+    @exception_wrap
```

Comment:
Shouldn't the `except requests.Timeout` also move into `exception_wrap`?

Reply:
The library no longer raises `requests.Timeout` (as of many releases ago) so this except check is unnecessary.  I've removed it from here and from anywhere else it was being caught.